chsdi/models/vector/stopo.py

```diff
@@ -1878,6 +1878,26 @@ class GeologieRohstoffeZementAbbauVerarbeitung(Base, Vector):
 register('ch.swisstopo.geologie-rohstoffe-zement_abbau_verarbeitung', GeologieRohstoffeZementAbbauVerarbeitung)
 
 
+class GeologieRohstoffeZiegelVerarbeitung(Base, Vector):
+    __tablename__ = 'rohstoffe_ziegel_verarbeitung'
+    __table_args__ = ({'schema': 'geol', 'autoload': False})
+    __bodId__ = 'ch.swisstopo.geologie-rohstoffe-ziegel_verarbeitung'
+    __template__ = 'templates/htmlpopup/rohstoffe_ziegel_verarbeitung.mako'
+    __queryable_attributes__ = ['obname', 'pckind']
```

Comment:
these attributes are supposed to be searchable, so they have to be part of the detail column in sphinx, you have to remove the __queryable_attributes__ property, this is for the configuration of  the object information tool

![image](https://user-images.githubusercontent.com/5286659/68391982-0a0c6600-0169-11ea-9113-82d1c0fe7704.png)

Reply:
therefore you should prepare a sphinx pr with this layer.

Reply:
Already done, see [sphinx](https://github.com/geoadmin/service-sphinxsearch/pull/409)